articles/active-directory/develop/quickstarts/active-directory-android.md

```diff
@@ -0,0 +1,180 @@
+>
+>> [!TIP]
+>> To know the *Application ID*, go to Overview page.
+
+> [!div renderon="portal" class="sxs-lookup"]
+> ### Step 1: Configure your application
+> For the code sample for this quickstart to work, you need to add a reply URL as **msal{AppId}://auth** (where {AppId} is the application ID of your app).
+> > [!div renderon="portal" id="makechanges" class="nextstepaction"]
+> > [Make this change for me]()
+>
+> > [!div id="appconfigured" class="hidden"]
+> > ![Already configured](../../../../includes/media/active-directory-develop-common/green-check.png) Your application is configured with these attributes
+
+#### Step 2: Download your web server or project
+
+* [Download the Android Studio Project](https://github.com/Azure-Samples/active-directory-android-native-v2/archive/master.zip)
+
+#### Step 3: Configure your project
+
+1. Extract and open the project in Android Studio.
+1. Under **app** > **java** > **<i>{host}.{namespace}</i>**, open **MainActivity**.
+1. Replace the line starting with `final static String CLIENT_ID` with the following code:
+
+    ```java
+    final static String CLIENT_ID = "[Enter the application Id here]";
+    ```
+1. Open: **app** > **manifests** > **AndroidManifest.xml**.
+1. Add the following activity to the **manifest\application** node. This code snippet registers a **BrowserTabActivity** to allow the OS to resume your application after completing the authentication:
+
+    ```xml
+    <!--Intent filter to capture System Browser calling back to our app after Sign In-->
+    <activity
+        android:name="com.microsoft.identity.client.BrowserTabActivity">
+        <intent-filter>
+            <action android:name="android.intent.action.VIEW" />
+            <category android:name="android.intent.category.DEFAULT" />
+            <category android:name="android.intent.category.BROWSABLE" />
+    
+            <!--Add in your scheme/host from registered redirect URI-->
+            <!--By default, the scheme should be similar to 'msal[appId]' -->
+            <data android:scheme="msalEnter_the_Application_Id_here"
+                android:host="auth" />
+        </intent-filter>
+    </activity>
+    ```
+> [!div renderon="docs"]
+> 6. Replace `Enter_the_Application_Id_here` with the Application ID your application
+
+## More Information
+
+Read the following sections for more info about this quickstart.
+
+### MSAL
+
+MSAL ([com.microsoft.identity.client](http://javadoc.io/doc/com.microsoft.identity.client/msal)) is the library used to sign in users and request tokens used to access an API protected by Microsoft Azure Active Directory (Azure AD). You can use carthage to install it by adding the following in **Gradle Scripts** > **build.gradle (Module: app)** under **Dependencies**:
+
+```ruby  
+compile ('com.microsoft.identity.client:msal:0.1.+') {
```

Comment:
This should be changed to    

`implementation 'com.android.volley:volley:1.1.1'` 
 `implementation ('com.microsoft.identity.client:msal:0.1.3') {
        exclude group: 'com.android.support', module: 'appcompat-v7'
    }`